docs/source/schemas/tables_and_indices.rst

```diff
@@ -495,6 +495,17 @@ succeeded by a ``partitioner()`` call, in the exact same manner as for
 table rows. For more information, see the Partitioners subsection of
 Table Rows.
 
+.. note::
+
+    Internally, index rows are stored including a reference to the source column,
+    but this is stripped out in the generated code before being returned to the user.
+    Thus, if one uses a List of results returned from an index table (e.g. through ``getRowColumns``,
```

Comment:
We have a `Set` below, so it seems rude not to have a `List`. I'll change this quickly.

Reply:
agreed, thanks!